optaweb-vehicle-routing-docs/src/main/asciidoc/quickstart.adoc

```diff
@@ -28,12 +28,12 @@ However, if you have a binary distribution of OptaWeb Vehicle Routing, you only
 java -version
 ----
 
-. If necessary, install OpenJDK 8.
-* To install OpenJDK 8 on Fedora, enter the following command:
+. If necessary, install OpenJDK 11.
+* To install OpenJDK 11 on Fedora, enter the following command:
```

Comment:
If we want to slightly nudge people towards 15, it's `dnf install java-openjdk-latest-devel`.

```suggestion
* To install OpenJDK on Fedora, enter the following command:
```

Reply:
Is it safe? Do we guarantee that project always builds and runs with the latest Java? If we had latest (wildcard, not a hardcoded version) Java in the CI matrix this would totally OK.

But can we risk that, say, in a year someone follows this, installs Java 17 and runs into problems? A) We can use it as an early warning about issues with the latest Java if the users reports the issue. B) The user gives up and we lose them.

Reply:
Very good point. Even if we specified "latest" JDK in our CI, we still depend on someone updating the "latest" there (Jenkins, Travis, GHA) soon enough.

We should not use something we don't test.

Reply:
If someone uses a SNAPSHOT version, I guess they can expect some problems and we will actually appreciate if they report problems to us.
For a stable version, issues with the latest JDK should already be resolved.

Reply:
Plus, I usually update our CI a day or two after the new version of OpenJDK is released.

Reply:
@rsynek If we had literally `latest` Java in CI (which will always be the latest release, as opposed to `15`) then it would be OK in my opinion.

Also, with the 6 month release cadence [1] I can imagine we'll get used to watching the releases and update our CI matrix regularly.

What is the chance of something actually breaking with a new Java release? I think it's highly unpredictable. The code should probably always keep working if you use the right dependencies. The build tools might break. We've had issues when upgrading to Java 11.

[1] https://www.oracle.com/java/technologies/java-se-support-roadmap.html